package.json

```diff
@@ -20,7 +20,7 @@
     "AWS"
   ],
   "author": "Wyatt Pearsall <wyatt.pearsall@cfpb.gov>",
-  "license": "CC0",
+  "license": "MIT",
```

Comment:
Yep, this was switched back in #23. Turns out the valid license signature for CC0 is CC0-1.0.

I think I squeezed this in because the warning was causing issues in a test environment, but it's fixed now